src/components/sbb-menu/sbb-menu.scss

```diff
@@ -86,10 +93,10 @@
   min-width: var(--sbb-menu-min-width);
   text-align: start;
   position: var(--sbb-menu-position);
-  left: 0;
-  bottom: 0;
-  top: unset;
-  right: unset;
+  inset-inline-start: 0;
+  inset-block-start: unset;
+  inset-block-end: 0;
+  inset-inline-end: unset;
```

Comment:
Unfortunately, this shorthand would not work because apparently 'unset' is not supported there. Where possible I replaced it with something like `inset: 0 auto auto 0`, so I can try using this but I'm not sure if it has the same effect.

Reply:
No, then it is fine as it is. I was not aware of that limitation. 😞